rtb/norme_di_progetto_ver0.6.0.typ

```diff
@@ -531,9 +556,177 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+- *Milestone*: la _release_ in cui deve essere pubblicato il risultato dell'attività;
+- *Etichetta*: la categoria a cui appartiene quella attività;
+- *Stato*: avanzamento del _task_.
+Ogni qualvolta ci sia la necessità di portare a termine un compito è necessario seguire la seguente procedura:
++ Il responsabile, dopo aver concordato in una riunione il da farsi, crea una nuova issue con stato "_to do_" su GitHub e la assegna;
++ All'inizio del lavoro di produzione, l'assegnatario cambia la _issue_  stato, passando da "_to do_" ad "_in progress_", inoltre crea un nuovo branch apposito staccato dal ramo principale per eseguire l'attività;
++ Contestualmente l'assegnatario apre una _pull request_ su GitHub in stato _draft_, per rendere pubblico il suo lavoro ancora prima che sia pronto. Nella descrizione della _pull request_ è importante che scriva la parola chiave "closes `#X`" (dove X è l'id della _issue_) per chiudere la _issue_ una volta che la _pull request_ sarà approvata e verrà effettuato il _merge_ nel ramo principale;
++ Finito il lavoro di produzione, viene modificata la _pull request_ dallo stato _draft_ allo stato _ready for review_ e viene assegnato il verificatore;
++ Il verificatore si controlla il lavoro svolto e:
+  - Se la verifica ha esito positivo:
+    + Il verificatore approva su GitHub la _pull request_ e chiede al responsabile di effettuare il _merge_ nel ramo principale ;
+    + La _issue_ viene marcata "_done_" su GitHub automaticamente.
+  - Se la verifica ha esito negativo:
+    + Il verificatore rilascia una lista di cambiamenti suggeriti nella _pull request_;
+    + L'incaricato apporta le modifiche suggerite e si torna al punto 4.
+
+=== Coordinamento
+Il coordinamento è l'attività che riguarda la gestione delle comunicazioni e degli incontri tra i vari membri del _team_, proponenti e committenti del progetto. L'efficienza del progetto e il coinvolgimento di tutte le parti interessate dipendono dal coordinamento.\
+La gestione della comunicazione interna ed esterna, la conduzione delle riunioni e la definizione di comportamenti comuni per i membri del _team_ sono esempi di attività di coordinamento.
+
+==== Comunicazioni
+===== Comunicazioni interne
+Le comunicazioni saranno gestite usando Telegram e Discord. Telegram, un servizio di messaggistica istantanea, sarà impiegato per conversazioni rapide, informali e accessibili tramite dispositivi mobili. Le riunioni a distanza e le discussioni più strutturate saranno invece condotte su Discord.\
+Al fine di coordinare in modo efficace le attività di gruppo, le discussioni di routine avverranno su Telegram, mentre eventuali questioni critiche saranno affrontate durante incontri straordinari attraverso videochiamate su Discord.\
+
+===== Comunicazioni esterne
+Il responsabile del progetto sarà incaricato di gestire il dialogo esterno attraverso l'indirizzo email: `sweat.unipd@gmail.com`.
```

Comment:
- toglierei i ":" dopo email
- "email" da mettere in corsivo
- togliere il monospace perché non è nome di un file, forse sono più indicate le virgolette oppure anche non mettere nulla.